### DISCUSSION TITLE
Help setting up a CKAN demo

### DISCUSSION THREAD

**Discussion by sheldonrampton** [ckan/ckan#8599] [Q&A]
I am on the board of directors of a nonprofit organization that compiles datasets related to climate change, and I am interested in building an open data portal for climate-related datasets. I have been trying to set up a proof-of-concept CKAN site that I can share with our staff for their review, but I have run into some snags, and I'm hoping someone can help me.

I first tried creating a CKAN site based on an AWS EC2 AMi from Link Digital -- the "Link Digital CKAN-2.11.0_20240826-prod-rs3ihy5fxynxy" AMI. That worked fine for creating a site and adding some custom content including organizations and a sample dataset, but I was unable to get the harvest feature working. (I want to show our staff a demo of CKAN's ability to harvest datasets from external sources.) I tried installing the Project Open Data extension so I could harvest a data.json source, but that broke the website, and my attempts at troubleshooting the errors have been unsuccessful.

I then tried creating a CKAN site in AWS by doing a package install on an EC2 instance running Ubuntu 22.04 and following the package install documentation I found here:

https://docs.ckan.org/en/2.11/maintaining/installing/install-from-package.html

After doing the install, however, the site has been throwing 504 errors. Again, I've tried troubleshooting those errors but have reached a point where I think I could benefit from some help. Is there someone who can help me?

**Comment by thegostev** [comment]
Hi, @sheldonrampton it's exciting that you'd be demoing CKAN. I have all faith that the tech issues are fixable as they always are. The downside of the current timing is that most of the contributors are on vacations. I'd made a call but not sure it'll be someone right now to help. May you duplicate the question in the Gitter chat where tech issues are tackled? It may help with higher exposure: https://app.gitter.im/#/room/#ckan_chat:gitter.im

Personally I'd be happy to follow up with your journey and help along the way.

**Comment by sheldonrampton** [comment]
Thanks, Alex. I'll post my question in the Gitter chat.

**Reply by pwalsh** [reply]
Hey @sheldonrampton - feel free to email me (email in GitHub profile) and I can make some time to do a video chat with you over the next week - I'm working over the period.

**Reply by sheldonrampton** [reply]
Hi, Paul. Thanks for your reply. I think my plans have changed a bit since I posted this request for help. I've discovered that someone has already built an open data portal for climate change-related datasets, so the project I had in mind might be reinventing the wheel. The portal that already exists was built with CKAN and hosts more than 25,000 datasets:

https://data.subak.org/

Rather than build a new portal, therefore, I think we'll probably focus on trying to help improve the visibility and quality of the portal that already exists.

--Sheldon

**Reply by pwalsh** [reply]
No worries!
